.travis.yml

```diff
@@ -1,12 +1,9 @@
 language: php
 
 php:
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
   - 7.1
   - 7.2
+  - 7.3
```

Comment:
7.4 also can be added, it is now available on Travis

Reply:
Done

Reply:
> Done

You may solve the conversation